src/Utilities/Compiler/Extensions/IOperationExtensions.cs

```diff
@@ -305,7 +305,7 @@ public static bool IsOperationNoneRoot(this IOperation operation)
             if (operation.Instance == null &&
                 operation.Property.ContainingType.IsAnonymousType)
             {
-                var declarationSyntax = operation.Property.ContainingType.DeclaringSyntaxReferences[0].GetSyntax();
+                var declarationSyntax = operation.Property.ContainingType.DeclaringSyntaxReferences[0].GetSyntax(); // TODO: Should the method take CancellationToken and pass it to GetSyntax() ?
```

Comment:
Possibly, but I would leave that out of this PR and revert the change in this file.